SRC/seleniumdemotest.py

```diff
@@ -1,48 +1,68 @@
 from selenium import webdriver
```

Comment:
seleniumdemotest name is good for now but maybe you could think of a more describing name for this file ?